demos/demo-react-ts/package.json

```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "build": "webpack",
     "build:gh": "npm ci && npm run build",
-    "serve": "webpack-dev-server --open",
-    "start": "npm run build && npm run serve",
+    "start": "npm run build && webpack-dev-server --open",
```

Comment:
To test this, delete the `dist` folder and run `npm start`, the compiled files will be generated and served by webpack on localhost.